src/ziptuple.rs

```diff
@@ -108,7 +108,7 @@ macro_rules! impl_zip_iter {
             #[inline]
             fn next_back(&mut self) -> Option<Self::Item> {
                 let ($(ref mut $B,)*) = self.t;
-                let size = *[$( $B.len(), )*].into_iter().min().unwrap();
```

Comment:
I think you're inadvertently reverting #479. Could you rebase?

Reply:
Sure thing!